From 6c0d57405831fdf51e778505000af3466a42af90 Mon Sep 17 00:00:00 2001
From: Jan Synacek <jsynacek@redhat.com>
Date: Mon, 8 Sep 2014 13:32:04 +0200
Subject: [PATCH] fix frontend config

---
 servers/slapd/slapd.ldif | 1 +
 1 file changed, 1 insertion(+)

diff --git a/servers/slapd/slapd.ldif b/servers/slapd/slapd.ldif
index 6c7c43c..18549d8 100644
--- a/servers/slapd/slapd.ldif
+++ b/servers/slapd/slapd.ldif
@@ -47,6 +47,7 @@ include: file://%SYSCONFDIR%/schema/core.ldif
 #
 dn: olcDatabase=frontend,cn=config
 objectClass: olcDatabaseConfig
+objectClass: olcFrontendConfig
 olcDatabase: frontend
 #
 # Sample global access control policy:
-- 
1.9.3

